posts/2024-04-15_gno-tokyo/README.md

```diff
@@ -0,0 +1,65 @@
+---
+publication_date: 2024-04-15T00:00:00Z
+slug: gno-tokyo
+tags: [meetup, tokyo, community]
+authors: [leohhhn]
+---
+    
+# Intro to Gno Recap - Tokyo 2024
+
+[![Banner](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/banner.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/banner.png)
+
+A Gno.land meetup, "Intro to Gno", was held in Tokyo, Japan on the 11th of 
+April 2024. Attendees joined the meetup in one of Tokyo's largest centers, 
+Shinjuku City. The meetup was aimed at a local audience, with the slides being
+in Japanese. 
+
+The meetup focused on explaining Gno.land's major features:
+- Fully open-source smart-contracting
+- Custom smart contracting language for the future
+- Proof of Contribution
+
+[![points](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/three-points.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/three-points.png)
+#### Brief overview of gno.land in Japanese
+
+Gno.land's DevRel Engineer, Leon, covered these three topics separately, discussing the importance of such 
+features in open-source blockchain systems.
+
+[![leon-poc](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/leon-poc.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/leon-poc.png)
+#### @leohhhn with the PoC explanation slide in Japanese
+The Gno.land ecosystem was also introduced: Leon mentioned some of the major 
+projects in the Gno.land ecosystem - Gnoswap, Gnoscan, Adena, Gnosocial, and
+more.
+
+Following this, attendees were introduced to the Gno.land 
+[**Grants & Fund program**](https://github.com/gnolang/ecosystem-fund-grants),
+as well as a the **Game of Realms** program, presenting different ways builders 
+are incentivized to build in the ecosystem at this stage.
+
+Finally, attendees were given insight into the status of the project:
+- Testnet 4, the final testnet before Gno.land Mainnet is scheduled to launch
+end of Q2
+- The Gno.land team is aiming to launch Mainnet by the end of 2024
+
+[![screens](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/screens.jpg)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/screens.jpg)
+#### Intro to Gno banner on venue screens
+[![merch](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/merch.jpg)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/merch.jpg)
+#### Gno.land keychains & stickers
+
+After the presentation, attendees enjoyed networking with their peers along with
```

Comment:
We connected with attendees to learn more about their projects and interests to get a better understanding of community, and get a sense of their takeways on Gno.land. (Did they have questions, what were some of the points they didn't understand....etc)